apps/asap-server/src/controllers/calendars.ts

```diff
@@ -91,6 +102,51 @@ export default class Calendars implements CalendarController {
     return res.items[0];
   }
 
+  fetchById(id: string, options?: { raw: false }): Promise<CalendarResponse>;
+  fetchById(id: string, options?: { raw: true }): Promise<CalendarRaw>;
+  async fetchById(
+    calendarId: string,
```

Comment:
well we could for consistency but i think i would like us to stop using generic ID name since it's confusing. Sometimes its the calendar ID, sometimes some other entity, and then there is an ID attached by squidex or google so